buildenvs/qemu.Dockerfile

```diff
@@ -1,100 +1,201 @@
+        --disable-lzfse \
+        --enable-lzo \
+        --disable-malloc-trim \
+        --disable-membarrier \
+        --disable-modules \
+        --disable-mpath \
+        --disable-multiprocess \
+        --disable-netmap \
+        --disable-nettle \
+        --disable-numa \
+        --disable-nvmm \
+        --disable-opengl \
+        --disable-oss \
+        --disable-pa \
+        --disable-parallels \
+        --disable-pie \
+        --disable-png \
+        --disable-profiler \
+        --disable-pvrdma \
+        --disable-qcow1 \
+        --disable-qed \
+        --disable-qga-vss \
+        --disable-rbd \
+        --disable-rdma \
+        --disable-replication \
+        --disable-safe-stack \
         --disable-sdl \
-        --disable-bluez \
+        --disable-sdl-image \
+        --disable-seccomp \
+        --disable-selinux \
+        --disable-slirp-smbd \
+        --disable-smartcard \
+        --disable-snappy \
+        --disable-sparse \
         --disable-spice \
+        --disable-spice-protocol \
+        --disable-tcg \
+        --enable-tools \
+        --disable-tpm \
+        --disable-u2f \
+        --disable-usb-redir \
+        --disable-user \
+        --disable-vde \
+        --disable-vdi \
+        --disable-vduse-blk-export \
+        --disable-vfio-user-server \
+        --enable-vhost-crypto \
+        --enable-vhost-kernel \
+        --enable-vhost-net \
+        --enable-vhost-user \
+        --enable-vhost-user-blk-server \
+        --enable-vhost-vdpa \
+        --disable-virglrenderer \
+        --disable-virtiofsd \
+        --disable-vmnet \
         --disable-vnc \
-        --disable-curses \
-        --disable-smartcard \
-        --disable-libnfs \
-        --disable-libusb \
-        --disable-glusterfs \
+        --disable-vnc-jpeg \
+        --disable-vnc-sasl \
+        --disable-vte \
+        --disable-vvfat \
         --disable-werror \
-        --target-list="x86_64-softmmu,i386-softmmu,aarch64-softmmu,arm-softmmu"; \
-    make; \
-    make install
+        --disable-whpx \
+        --${WITH_XEN}-xen \
+        --${WITH_XEN}-xen-pci-passthrough \
+        --disable-xkbcommon \
+        --disable-zstd \
+        --enable-virtfs \
+        ; \
+        make -j${MAKE_NPROC}; \
+        make install;
 
 FROM scratch AS qemu
 
-COPY --from=qemu-build /bin/qemu-ga /bin/
-COPY --from=qemu-build /bin/qemu-img /bin/
-COPY --from=qemu-build /bin/qemu-io /bin/
-COPY --from=qemu-build /bin/qemu-nbd /bin/
-COPY --from=qemu-build /bin/qemu-pr-helper /bin/
-COPY --from=qemu-build /bin/qemu-system-aarch64 /bin/
-COPY --from=qemu-build /bin/qemu-system-arm /bin/
-COPY --from=qemu-build /bin/qemu-system-i386 /bin/
-COPY --from=qemu-build /bin/qemu-system-x86_64 /bin/
+COPY --from=qemu-build /bin/qemu-system-x86_64 /bin/qemu-g[a] \
+    /bin/qemu-system-i38[6] /bin/qemu-system-ar[m] /bin/qemu-system-aarch6[4] \
+    /bin/qemu-pr-helpe[r] /bin/qemu-im[g] /bin/qemu-i[o] /bin/qemu-nb[d] /bin/
```

Comment:
I believe this change was done to reduce the number of copy steps, right? I think because all other Dockerfiles are splitting things into new lines with a `\` separator we should do the same and we should be explicit with what we're copying, plus it makes diffs in the future easier to blame:
```diff
- COPY --from=qemu-build /bin/qemu-system-x86_64 /bin/qemu-g[a] \
-     /bin/qemu-system-i38[6] /bin/qemu-system-ar[m] /bin/qemu-system-aarch6[4] \
-     /bin/qemu-pr-helpe[r] /bin/qemu-im[g] /bin/qemu-i[o] /bin/qemu-nb[d] /bin/
+ COPY --from=qemu-build /bin/qemu-ga
+                        /bin/qemu-img \
+                        /bin/qemu-io \
+                        /bin/qemu-nbd \
+                        /bin/qemu-pr-helper \
+                        /bin/qemu-system-aarch64 \
+                        /bin/qemu-system-arm \
+                        /bin/qemu-system-x86_64 \
+                        /bin/qemu-system-i386 \
+                        /bin/qemu-system-x86_x66 \
+                        /bin/
```